descriptors/ModuleDescriptor-template.json

```diff
@@ -153,7 +153,7 @@
     },
     {
       "id": "instance-storage",
-      "version": "8.0",
+      "version": "8.1",
```

Comment:
Is this the interface that uses this schema? No bulk edits or other APIs that use it?

Reply:
I ran a search for the file name on the entire repo-this is the only interface/raml that directly references instance.json.  There are other instance-related endpoints, but they reference separate instance representations (like inventory-view-instance.json), which I have not changed.  Other json files reference the file (like instances.json), but I don't think their interfaces/ramls need to change?

Reply:
> I ran a search for the file name on the entire repo-this is the only interface/raml that directly references instance.json. There are other instance-related endpoints, but they reference separate instance representations (like inventory-view-instance.json), which I have not changed.

Thank you for doing that.

> Other json files reference the file (like instances.json), but I don't think their interfaces/ramls need to change?

I believe `instances.json` references `instance.json` meaning they would need to change.

Reply:
So anything that references a json which references instance.json would need to increment?  Because that's a lot more ramls/interfaces.

Reply:
> So anything that references a json which references instance.json would need to increment? Because that's a lot more ramls/interfaces.

Yes, because that means the change is included in the interface definition.